gradle/dependencies.gradle

```diff
@@ -13,7 +13,7 @@ def versions = [
     kotlinPluginMin     : '1.4.0',
     kotlinCoroutines    : '1.5.2',
     moshi               : '1.12.0',
-    moshix              : '0.14.1',
+    moshix              : '0.13.0',
```

Comment:
oh no! No new moshix for us?

Reply:
`moshi-sealed-ksp` didn't exist for `0.14.1` so I just reverted, but actually it's normal, it is because it has been [merged](https://github.com/ZacSweers/MoshiX/releases#:~:text=moshi%2Dsealed%2Dksp%20has%20now%20been%20merged%20into%20moshi%2Dsealed%2Dcodegen) with `moshi-sealed-codegen`. I changed the dependency in the next commit and now we can use the latest version.